src/main/kotlin/graphics/scenery/volumes/TransferFunctionEditor.kt

```diff
@@ -353,6 +362,60 @@ class TransferFunctionEditor constructor(
         rangeEditorPanel.add(maxValueLabel, "spanx 2, right")
 
 //        updateSliderRange()
+
+        //ColorMap manipulation
+        colormapPanel = JPanel()
+        colormapPanel.layout = MigLayout("fill",
+            "[left, 10%]5[right, 40%]5[left, 10%]5[right, 40%]")
+        add(colormapPanel, "grow")
+
+        val list = Colormap.list()
+        val box = JComboBox<String>()
+        for (s in list)
+            box.addItem(s)
+
+        if (tfContainer is DummyVolume){
```

Comment:
This code can't be here as it ties the TransferFunction editor to VDI rendering in a relatively ugly way.